website/pages/gamejam_en.md

```diff
@@ -10,91 +10,124 @@ This will be followed by some play testing of old game jam games while chilling
+- Global Game Jam Linköping 2021 - *"Lost and Found"* - 
+[Itch.io](https://itch.io/jam/global-game-jam-linkoping-2021), [Global Game Jam](https://globalgamejam.org/2021/jam-sites/liu-game-jam),
+ [Intro](https://www.youtube.com/watch?v=yKhhIuSa49M), [Presentation](https://www.youtube.com/watch?v=lEk12BYQYgo)
+
+### 2020
+
+- Fall Game Jam 2020 - *"Distance Plain", "Giant 2020"* - [Itch.io](https://itch.io/jam/liu-fall-game-jam-2020), 
+ [Intro](https://www.youtube.com/watch?v=wJ1pcaT8opk), [Presentation](https://www.youtube.com/watch?v=MpBZgX6GBuY)
+- Spring Game Jam 2020 - *"Orange Expansions", "Ghoul Reassembly"* - [Itch.io](https://itch.io/jam/liu-spring-game-jam-2020), 
+ [Intro](https://www.youtube.com/watch?v=yKhhIuSa49M), [Presentation](https://www.youtube.com/watch?v=MpBZgX6GBuY)
+- Global Game Jam Linköping 2020 - *"Repair"* - [Global Game Jam](https://globalgamejam.org/2020/jam-sites/liu-game-jam-ggj-2020)
+
+### 2019
+
+- Fall Game Jam 2019 - *"Wild West Planes, but with eyes", "Compost War: Two worlds, one truck", "Musical tentacles"* - [Presentation](https://www.facebook.com/349988561805129/videos/799393110511100)
+- Spring Game Jam 2019 - *"Spinning Change", "GODZILLA In Real Life", "Very large Pizza cutter""* - [Presentation](https://www.facebook.com/349988561805129/videos/297587374524542)
+- Global Game Jam Linköping 2019 - *"What home means to you"* - [Global Game Jam](https://globalgamejam.org/2019/jam-sites/liu-game-jam/games)
+
+### 2018
+
+- Fall Game Jam 2018 - *"Metal Time travel", "Navy Nightmare", "Big sound House"*
+- Spring Game Jam 2018 - *"Extreme-dancing King", "Coyote Solitude"* - [Presentation](https://www.facebook.com/349988561805129/videos/1279817882155521)
+- Global Game Jam 2018 Linköping - *"Transmission"* - [Global Game Jam](https://globalgamejam.org/2018/jam-sites/liu-game-jam)
+
+### 2017
+
+- Fall Game Jam 2017 - *"Royal Styling", "Flat abuse", "Speedy Cactus"*
+- Spring Game Jam 2017  - *"Highschool Race", "Icecream monsters", "Pixel Erasers"*
+- Global Game Jam 2017 Linköping - *"Waves"* - [Global Game Jam](https://globalgamejam.org/2017/jam-sites/liu-game-jam)
+
+### 2016
+
+- Fall Game Jam 2016 - *"Duck Crabs", "Chocolate Guitars", "Angry monkeys (but with more legs)"*
+- Spring Game Jam 2016 - *"Zombie Feelings", "Pink Song-contest", "Mountain Chicken"*
+- Global Game Jam 2016 - *"Ritual"* - [Global Game Jam](https://globalgamejam.org/2016/jam-sites/liu-game-jam)
+
+### 2015
+
+- Fall Game Jam 2015 - *"Roundabout Battle", "Raging Change"*
+- Spring Game Jam 2015 - *"Disruptive rabbits"*
+- Linköping Global Game Jam 2015 - *"What do we do now"* - [Global Game Jam](https://globalgamejam.org/2015/jam-sites/liu-game-jam)
+
+### 2014
+
+- Fall (of the) Game Jam 2014 - *"Rocket Castle"*
+- Global Game Jam 2014 - *"We don't see things as they are, we see them as we are"*
+
+### 2013
+
+- Spring Game Jam 2013
+
+### 2012
+
+- Global Game Jam 2012 - *"Ouroboros"* 
+
+---
```

Comment:
Jag gillar idéen! Men man kanske ska ha en hel sida till det? Är saker lätta att hitta när ligger i en sida?

Reply:
Det skulle i så fall bli en till sida i sidomenyn antar jag? Det skulle vara mycket möjligt under förutsättning att lithe-kod är OK med det.

Reply:
En ide är att ha en table of contents och en "return to table of contents" vid varje rubrik. Just nu kan det bli lite överväldigande att öppna sidan första gången.

Reply:
> 
> 
> Jag tycker att h-strecken gör sidan lite mer förvirrad än vad den behöver vara. Det läses mer som inlägg än som olika ämnen så att säga. Eftersom varje sektion börjar med en rubrik känns det som om sidan ändå blir tillräckligt uppdelad även utan strecken.

Förståligt, vad tycker övriga?

Tänker att vi nog kan vilja ha en avgränsning mellan "senaste jam info" och mera generell game jam / FAQ info.

Reply:
> 
> 
> En ide är att ha en table of contents och en "return to table of contents" vid varje rubrik. Just nu kan det bli lite överväldigande att öppna sidan första gången.

Har vi stöd för detta på hemsidan? 
Spontakt tänker jag att om game jam sidan är den endas som har en table of content så bör den delas upp.

Reply:
Table of contents gissar jag att man gör med id:n, men jag tror den riktiga lösningen är att hålla det på en lagom nivå.

Sedan med h-streck tror jag att det är bättre utan, och det skulle inte vara helt fel med den gula färgen. Vad tycker ni?
![image](https://user-images.githubusercontent.com/16568066/115278513-d4352480-a145-11eb-86e9-48bb22c54abb.png)

```html
<style>
    h1, h2, h3 {
        color: #ffc600;
    }
</style>
```
I början av filen

Reply:
Gula var snyggt fast jag tycker det bör stämma med övriga lithe kod sidan.
(hur ser det ut i dags-tema?)

Reply:
Vi har diskuterat just den gula färgen och ljust tema förut. Jag kommer inte ihåg vad vi sa men det går att leta upp i protokollen, men:

Vid det här laget tycker jag att vi mergar den här texten som den är (eller utan hr, för min del spelar det ingen roll vilken) och diskuterar uppdelning av sidor och gula rubriker separat. Ingen verkar ha några åsikter om innehållet i texten och markdown är jättebra.